app/scripts/metamask-controller.test.js

```diff
@@ -730,6 +730,10 @@ describe('MetaMaskController', () => {
     describe('unlockHardwareWalletAccount', () => {
       const accountToUnlock = 10;
       beforeEach(async () => {
+        await metamaskController.coreKeyringController.createNewVaultAndRestore(
+          'password',
+          TEST_SEED,
+        );
```

Comment:
We can't really test anything properly without a vault and an unlocked `KeyringController`.